### PR TITLE
Patch 14

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ manager: serdars
 
 # Manage user mailboxes
 
-After you create a user mailbox, you can make changes and set additional properties by using the EAC or Exchange Online PowerShell.
+After you create a user mailbox, you can make changes and set additional properties by using the new Exchange admin center(EAC) or Exchange Online PowerShell.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:25]:
```suggestion
After you create a user mailbox, you can make changes and set additional properties by using the new Exchange admin center (EAC) or Exchange Online PowerShell.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 2]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
 > Having problems? Ask for help in the Exchange forums. Visit the forums at [Exchange Online](https://social.technet.microsoft.com/forums/msonline/home?forum=onlineservicesexchange) or [Exchange Online Protection](https://social.technet.microsoft.com/forums/forefront/home?forum=FOPE).
 
 ## Change user mailbox properties
+### Configure user mailbox properties in the new EAC
 
-### Use the EAC to change user mailbox properties
+1. In the new EAC, navigate to **Recipients** \> **Mailboxes**.
+The **mailboxes** and **shared** tabs under **Recipients** (of the classic EAC) are now merged into a single tab, **Mailboxes** in the new EAC. On clicking the **Mailboxes** tab, you can view the shared and user mailboxes under one list view. On top of the list of shared and user mailboxes, the following options are provided:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:42]:
```suggestion
The **mailboxes** and **shared** tabs under **Recipients** (of the classic EAC) are now merged into a single tab, **Mailboxes**, in the new EAC. On clicking the **Mailboxes** tab, you can view the shared and user mailboxes under one list view. On top of the list of shared and user mailboxes, the following options are provided:
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 3]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+
+   - **Refresh**: Use this option to refresh the mailbox list.
+
+   - **Export**: Use this option to download a .csv file(excel sheet) with details of all the user mailboxes.
+
+   - **Search**: Use this option to search for any user mailbox by entering the suitable keyword.
+
+   - **Filter**: Use this detailed option for creating custom filters or using shared filters.
+
+   - **Normal List** and **Compact List** - The default view that you see when you open Mailboxes is the normal listing view. In the Compact List view you can see more number of columns with reduced spaces in between.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:55]:
```suggestion
   - **Normal List** and **Compact List** - The default view that you see when you open Mailboxes is the normal listing view. In the Compact List view, you can see more columns with reduced spaces in between.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 4]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+   
+   - Also, when you select any mailbox by clicking on the radio button next to the display name(on any row), certain additional options are also available on top:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:57]:
```suggestion
   - Also, when you select any mailbox by clicking on the radio button next to the display name (on any row), certain additional options are also available on top:
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 5]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+   
+      - **Hide from address list**: Select this option to prevent the recipient from appearing in the address book and other address lists that are defined in your Exchange organization. After you select this option, users can still send messages to the recipient by using the email address.
+      
+      - Other additional options such as Edit contact information, Manage mailbox delegation, Recover deleted items, Convert to shared mailbox, Edit email address, Refresh etc. are also displayed on clicking the more options (**...**) button on top.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:61]:
```suggestion
      - Other additional options such as Edit contact information, Manage mailbox delegation, Recover deleted items, Convert to shared mailbox, Edit email address, Refresh, etc. are also displayed on clicking the more options (**...**) button on top.  
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 6]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+   
+      - **Hide from address list**: Select this option to prevent the recipient from appearing in the address book and other address lists that are defined in your Exchange organization. After you select this option, users can still send messages to the recipient by using the email address.
+      
+      - Other additional options such as Edit contact information, Manage mailbox delegation, Recover deleted items, Convert to shared mailbox, Edit email address, Refresh etc. are also displayed on clicking the more options (**...**) button on top.  
+
+2. In the list of user mailboxes, click the mailbox that you want to change the properties for. A flyout is displayed for the selected user mailbox.
+
+3. On this page, the **Mailbox** and **Account** links are provided for changing the properties. 
+
+4. Use the **Mailbox** link to change any of the following the properties.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:67]:
```suggestion
4. Use the **Mailbox** link to change any of the following properties:
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 7]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+**Add email address type**: Click ![Add Icon](../../media/ITPro_EAC_AddIcon.gif) **Add email address type** to add a new email address for this mailbox. Select one of following address types:
+
+   - **SMTP**: This is the default address type. Click this button and then type the new SMTP address in the **Email address\*:** box.
+
+   - **Enter a custom address type**: Click this button and type one of the supported non-SMTP email address types in the **Email address\*:**  box.
+
+     > [!NOTE]
+     > With the exception of X.400 addresses, Exchange doesn't validate custom addresses for proper formatting. You must make sure that the custom address you specify complies with the format requirements for that address type.
+
+   - **Make this the reply address**: In Exchange Online, you can select this check box to make the new email address the primary SMTP address for the mailbox. This check box  isn't available in the EAC in Exchange Server. Click OK to save your changes.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:101]:
```suggestion
   - **Make this the reply address**: In Exchange Online, you can select this check box to make the new email address the primary SMTP address for the mailbox. This check box isn't available in the EAC in Exchange Server. Click OK to save your changes.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 8]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+
+- **Read and manage**: This permission allows a delegate to sign in to the user's mailbox and view the contents of the mailbox. However, after this permission is assigned to a delegate, the delegate can't send messages from the mailbox. To allow a delegate to send email from the user's mailbox, you still have to assign the delegate the Send As or the Send on Behalf Of permission.
+
+To assign permissions to delegates click on the **Edit** button next to the appropriate permission. Next, click ![Add Icon](../../media/ITPro_EAC_AddIcon.gif) **Add permissions** . This displays a list of all recipients in your Exchange organization that can be assigned the permission. Select the recipients you want, add them to the list, and then click **Save**. You can also search for a specific recipient by typing the recipient's name in the search box.
+
+#### Mailbox policies
+Use the **Mailbox policies** section to apply default mailbox policies for the organization. Click **Manage mailbox policies** under **Mailbox policies**. A flyout is displayed and you can view or change the mailbox policies. Click ![Edit icon](../../media/ITPro_EAC_EditIcon.gif) and change the following mailbox policies.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:124]:
```suggestion
Use the **Mailbox policies** section to apply default mailbox policies for the organization. Click **Manage mailbox policies** under **Mailbox policies**. A flyout is displayed and you can view or change the mailbox policies. Click ![Edit icon](../../media/ITPro_EAC_EditIcon.gif) and change the following mailbox policies:
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 9]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+- **Sharing policy**: This box shows the sharing policy applied to the mailbox. A sharing policy controls how users in your organization can share calendar and contact information with users outside your Exchange organization. The Default Sharing Policy is assigned to mailboxes when they are created. To change the sharing policy that's assigned to the user, select a different one from the drop-down list.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:125]:
```suggestion
- **Sharing policy**: This box shows the sharing policy applied to the mailbox. A sharing policy controls how users in your organization can share calendar and contact information with users outside your Exchange organization. The default sharing policy is assigned to mailboxes when they are created. To change the sharing policy that's assigned to the user, select a different one from the drop-down list.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 10]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+  - **Note (visible to the user)**: Use this box to notify the user about the litigation hold, explain why the mailbox is on litigation hold, or provide additional guidance to the user, such as informing them that the litigation hold won't affect their day-to-day use of email.
+
+  - **Web page with more information for the user**: Use this box to provide a URL to a website that provides information or guidance about the litigation hold on the mailbox.
+
+    > [!NOTE]
+    > The text from these boxes appears in the user's mailbox only if they are using Outlook 2010 or later versions. It doesn't appear in Outlook on the web or other email clients. To view the text from the Note and URL boxes in Outlook, click the **File** tab, and on the **Info** page, under **Account Settings**, you'll see the litigation hold comment.
+    
+- **Manage mailbox archive**: Use this option to enable or disable the archive mailbox.
+
+- **Set recipient limit**: This setting controls the maximum number of recipients the user can send a message to. Specify the maximum number of recipients in the Maximum recipients text box . In Exchange Online, the limit is 500 recipients.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:155]:
```suggestion
- **Set recipient limit**: This setting controls the maximum number of recipients the user can send a message to. Specify the maximum number of recipients in the **Maximum recipients** text box. In Exchange Online, the limit is 500 recipients.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 11]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+
+- **Custom attributes**: Custom attributes are extension attributes that you can use to add information about a recipient for which there isn't an existing attribute. You can add a maximum of 15 custom attributes to a mailbox. Click ![Add Icon](../../media/ITPro_EAC_AddIcon.gif) **Add custom attribute** to add custom attributes.
+
+#### Automatic replies
+
+Use these settings to create automatic reply (Out of Office) messages. Click **Manage automatic replies** under **Automatic replies**. A flyout is displayed and you can turn ON the **Automatic replies** option.
+Specify the following information:
+- **Reply to all senders inside the organizations from this mailbox** - Enter the automatic reply message in this text box. 
+> [!Note] This field cannot be empty if automatic reply is on.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:166]:
```suggestion
> [!NOTE]
> This field cannot be empty if automatic reply is on.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 12]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+
+- **Send automatic replies to senders outside the organizations from this mailbox** - Enable this check box to send automatic replies to senders outside the organizations from this mailbox. On enabling this check box, you can choose between the options, **Only reply to senders in the mailbox's contact list** or **Reply to all sender**. Enter the automatic reply message in the **Reply to all senders outside the organizations from this mailbox** text box.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:168]:
```suggestion
- **Send automatic replies to senders outside the organizations from this mailbox** - Enable this check box to send automatic replies to senders outside the organizations from this mailbox. On enabling this check box, you can choose between the options, **Only reply to senders in the mailbox's contact list** or **Reply to all senders**. Enter the automatic reply message in the **Reply to all senders outside the organizations from this mailbox** text box.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 13]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+
+#### Email apps
+
+Use this section to apply the default settings for outlook for web, IMAP, POP3, MAPI applied. Click **Manage email apps settings** under **Email apps**. A flyout is displayed and you can set the default settings for the following:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:172]:
```suggestion
Use this section to apply the default settings for Outlook on the web, IMAP, POP3, and MAPI applied. Click **Manage email apps settings** under **Email apps**. A flyout is displayed and you can set the default settings for the following:
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md [hunk 14]
```diff
@@ -36,8 +36,155 @@ After you create a user mailbox, you can make changes and set additional propert
...
+- Outlook on the web
+- Outlook desktop (MAPI)
+- Exchange web services
+- Mobile (Exchange ActiveSync)
+- IMAP
+- POP
+
+> [!Note] All these fields are enabled by default.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/manage-user-mailboxes.md:180]:
```suggestion
> [!NOTE]
> All these fields are enabled by default.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**